### PR TITLE
Pantheon changes

### PR DESCRIPTION
**Pull request by stevector**
Hi @Schnitzel! Here are some changes for Pantheon.

### CODE REVIEW

**Comment by Schnitzel** [comment]
@stevector nice! thanks, will review and commit.

File: comparison.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Help improve our documentation! If you find anything wrong or which could be imp
 			<tr>
 				<td><strong>Custom web server configurations</strong></td>
 				<td>✅ Custom configurations possible. Easy testing inside Local Docker Development</td>
-				<td>❌️ No custom configurations</td>
+				<td>⚠️ Some customizability via pantheon.yml file</td>
```
Comment (Schnitzel) [inline comparison.md:32]: @stevector do you have an example what can be configured? I see that there is php version. but what else?

File: comparison.md [hunk 2]
```diff
@@ -442,7 +442,7 @@ Help improve our documentation! If you find anything wrong or which could be imp
 			<tr>
 				<td><strong>Support for external (Amazon S3, Dropbox, etc.) backups via “backup and migrate” module</strong></td>
 				<td>✅ Full support to save backups externally. Custom crons provide even more flexibility.</td>
-				<td>⚠️ “backup and migrate” module not recommended</td>
+				<td>⚠️ Backups made to S3 automatically. “backup and migrate” module not recommended</td>
```
Comment (Schnitzel) [inline comparison.md:445]: @stevector But this is the S3 account of Pantheon? So if Pantheon cannot pay the bills anymore (see Aberdeen), the client has no access to the backups?

File: comparison.md [hunk 3]
```diff
@@ -56,7 +56,7 @@ Help improve our documentation! If you find anything wrong or which could be imp
 			<tr>
 				<td><strong>24x7 infrastructure and application monitoring</strong></td>
 				<td>✅</td>
-				<td>⚠️ Only on Elite plan</td>
+				<td>✅ New Relic APM Pro free for all sites. Additional monitoring for Elite plan</td>
```
Comment (Schnitzel) [inline comparison.md:59]: we actually mean here that the application is monitored for the client by the Hoster, so I updated slightly that it's a self-serve system: https://github.com/amazeeio/docs/pull/16/commits/81732b9c926652ecb7eb83c5149c6ee352a15caf

File: comparison.md [hunk 4]
```diff
@@ -375,7 +375,7 @@ Help improve our documentation! If you find anything wrong or which could be imp
 			<tr>
 				<td><strong>New Relic</strong></td>
 				<td>✅ Free New Relic Pro (all production sites in one single account)</td>
-				<td>✅ Free New Relic Pro (each production site in separate accounts)</td>
+				<td>✅ Free New Relic Pro for all sites (including free pre-launch sites) and all environments (Each site in its own account)</td>
```
Comment (Schnitzel) [inline comparison.md:378]: commited! https://github.com/amazeeio/docs/pull/16/commits/81732b9c926652ecb7eb83c5149c6ee352a15caf

**Comment by dasrecht** [comment]
Closing as those changes are very old by now and might not be accurate anymore.

**Pull request closed without merging**